include/envoy/server/configuration.h

```diff
@@ -132,6 +132,27 @@ class Main {
    *         counters.
    */
   virtual std::chrono::milliseconds statsFlushInterval() PURE;
+
+  /**
+   * @return the time interval after which we count a nonresponsive thread event
+   * as a "miss" statistic.
```

Comment:
I don't think doxygen actually does anything special with that, but this is a good convention to follow.  I will update the existing line above (statsFlushInterval()) in the same way.